ui/src/containers/appView/SaveAppModalContainer.js

```diff
@@ -7,12 +7,12 @@ import { toggleSaveAppModal } from '../../actions/actions-ui';
 import ConfirmationModal from '../../components/common/ConfirmationModal';
 
 /**
- * Redux container for the Save App Modal
+ * Stateful container for the Save App Modal
  *
  * @param {boolean} props.isOpen - The open/closed state of the modal.
  * @param {string} props.title - The title to display on the modal.
- * @param {*} props.children - The child props of this component.
- * @param {*} props.onConfirm - The action to execute when `Confirm` is clicked.
+ * @param {node} props.children - The child props of this component.
```

Comment:
`node` isn't really a suitable type here IMO. `children` can technically be anything, from a `div` to a single word. Generally, this can be denoted with `*` for anything.